.github/workflows/publish.yml

```diff
@@ -0,0 +1,29 @@
+name: Publish Extension
+
+on:
+  workflow_run:
+    workflows: ["Test VsCode Extension"]
+    types:
+      - completed
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '18'
+
+      - name: Install dependencies
+        run: npm install
+      
+      - name: Publish to Visual Studio Marketplace
+        uses: HaaLeo/publish-vscode-extension@v1
+        with:
+          pat: ${{ secrets.VS_MARKETPLACE_TOKEN }}
```

Comment:
What the scopes of VS_MARKETPLACE_TOKEN and can it be replaced with GITHUB_TOKEN?

Reply:
This workflow is for publishing the extension to the vscode marketplace so that it can be used by users. As far as I know, this is uniquely generated by vscode marketplace and is not replacable with a github token